contrib/objcwrapper/tests/MyClass.h

```diff
@@ -0,0 +1,12 @@
+@interface MyClass : NSObject
+  <NSCoding, NSCopying, NSMutableCopying, NSFastEnumeration>
+
++ (id) array;
++ (id) arrayWithContentsOfFile: (NSString*)file;
+
+- (id) arrayWithArray: (MyClass*)array;
+
+@property (readonly) NSUInteger count;
+@property NSUInteger head;
+
+@end
```

Comment:
It this a made-up language for CSG? I like the use of - and +, they make diffs a delight to read.